assets/model_monitoring/components/src/action_analyzer/action_analyzer_metrics_calculation/run.py

```diff
@@ -50,6 +52,17 @@
 )
 
 
+def _post_process_results(output):
+    parsed_score_response = re.findall(r'\d+', output.split("# Result")[-1].strip())
+    if len(parsed_score_response) > 0:
+        score = float(parsed_score_response[0].replace("'", "").strip())
```

Comment:
Should we add try here

Reply:
I copied this exact parse function from the sdk. Basically, when successfully parsed (regex match), we set it as 1-5. If the result does not match the expected regex, we set it as 0. Then later when we do the T-test, we filter out 0s. 

In my test, 80 samples I saw only one 0 (failure). 

https://msdata.visualstudio.com/Vienna/_git/AzureMlCli?path=/src/azureml-metrics/azureml/metrics/text/rag_evaluation/_rag_evaluation.py&version=GBmaster&line=535&lineEnd=543&lineStartColumn=1&lineEndColumn=29&lineStyle=plain&_a=contents